src/main/java/htsjdk/samtools/cram/build/ContainerParser.java

```diff
@@ -59,60 +60,55 @@ public ContainerParser(final SAMFileHeader samFileHeader) {
         return records;
     }
 
-    public Map<Integer, AlignmentSpan> getReferences(final Container container, final ValidationStringency validationStringency) {
```

Comment:
someday we will merge a PR that collapses all these stupid things.  I may even have such a thing open?

Reply:
Yes: that's over at #1276